### PR TITLE
Clear state button

### PR DESCRIPTION
**Pull request by li-lucia**
## Overview

**Issue Type**

- [ ] Bug
- [x] Feature
- [ ] Tech Debt

**Description**
Current version includes the following changes:
1) function that abstracts otel data to only display relevant data
2) creates and updates only one state that both table and chart uses
3) functions that converts state data to chartjs compatible data
4) button that manually clears state
5) changes to chart size display
6) created reload div that renders when state is empty

**Screenshots & Videos**
Current display screenshot:

Screen Shot 2023-07-07 at 6 20 24 PM

### CODE REVIEW

File: chrome-extension/src/pages/app/app.tsx [hunk 1]
```diff
@@ -2,44 +2,55 @@ import { useEffect, useState } from 'react';
...
 function App() {
-  const [networkRequests, setNetworkRequests] = useState<OtelData[]>([]);
+  const [relevant, setRelevant] = useState<IRelevant>(new Map()); 
 
+
+  // testing uses only: to cross-reference incoming client data with otel data
   useEffect(() => {
     chrome.runtime.onMessage.addListener((message) => {
-      console.log('received chrome message', message)
+      // console.log(`chromeApiRequestData: ${message.type}, startTime ${message.startTime}`, message);
```
Comment (YouruiR) [inline chrome-extension/src/pages/app/app.tsx:17]: I noticed a lot of console logs that are still in the code, but commented out. Would these be fine to take out?

File: chrome-extension/src/pages/app/app.tsx [hunk 2]
```diff
@@ -2,44 +2,55 @@ import { useEffect, useState } from 'react';
...
   }, []);
 
 
-  useEffect(() => {
-    // console.log(networkRequests);
-  }, [networkRequests]);
- 
-  const aggregatedData = checkTraceId(networkRequests);
-
+  // useEffect(() => {
```
Comment (YouruiR) [inline chrome-extension/src/pages/app/app.tsx:34]: console log

File: chrome-extension/src/pages/app/functions/chartJSData.ts [hunk 3]
```diff
@@ -1,30 +1,23 @@
...
-    //   earliestTime = singleTraceData[0].start
-    // }
-    const firstVal = calcTotalTime(earliestTime, singleTraceData[0].start);
-  
-    const secondVal: number = firstVal + singleTraceData[0].duration;
-    singleChartJSData.push(firstVal, secondVal);
-    output.push(singleChartJSData);
-    // console.log('singleChartJSData', singleChartJSData);
+  allRequestData.forEach((request) => {
+    // console.log('relative start time',request.relativeStartTime);
```
Comment (YouruiR) [inline chrome-extension/src/pages/app/functions/chartJSData.ts:7]: console log

File: chrome-extension/src/pages/devtools/index.ts [hunk 4]
```diff
@@ -18,22 +14,25 @@ try {
...
-        chrome.runtime.sendMessage(networkObject);
-      }
-    )
+
+
+  chrome.devtools.network.onRequestFinished.addListener(
+    function(request) {
+      const networkObject: IChromeApiNetworkObject = {};
+      if (request.request.httpVersion === 'chrome-extension') return;
+      // console.log(request);
```
Comment (YouruiR) [inline chrome-extension/src/pages/devtools/index.ts:23]: console log

File: chrome-extension/src/pages/panel/index.tsx [hunk 5]
```diff
@@ -6,6 +6,6 @@ import App from '../app/app.tsx'
 const domNode: HTMLElement = document.getElementById('root')!
 const root = createRoot(domNode)
 
-console.log('panel/index.jsx reached')
+// console.log('panel/index.jsx reached')
```
Comment (YouruiR) [inline chrome-extension/src/pages/panel/index.tsx:9]: console log

**Review by YouruiR — APPROVED** [review]

**Review by davidan1989 — APPROVED** [review]

**Pull request merged**